src/catkin_pkg/package_templates.py

```diff
@@ -106,13 +112,23 @@ def _create_package_template(package_name, description=None, licenses=None,
             if dep.lower() == 'message_generation':
                 if not 'message_runtime' in catkin_deps:
                     sys.stderr.write('WARNING: Packages with messages or services should depend on both message_generation and message_runtime\n')
+                    catkin_deps.append('message_runtime')
```

Comment:
I am not sure if overriding the user input is the right thing here. This also doesn't happen for the `genmsg` case above.

Same below.